cmd/workspace.go

```diff
@@ -198,6 +203,85 @@ func newWorkspaceUserListCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+/*
+	Workspace teams commands
+*/
+
+func newWorkspaceTeamRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "team",
+		Short: "Manage Workspace Team resources",
+		Long:  "Teams can be added or removed from Workspaces",
+	}
+	cmd.AddCommand(
+		newWorkspaceTeamAddCmd(out),
+		newWorkspaceTeamUpdateCmd(out),
+		newWorkspaceTeamRmCmd(out),
+		newWorkspaceTeamListCmd(out),
+	)
+	return cmd
+}
+
+func newWorkspaceTeamAddCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "add TEAM",
+		Short:   "Add a Team to a Workspace",
+		Long:    "Add a Team to a Workspace",
+		Example: "astro workspace team add <team-id> --workspace-id <workspace-id> --role <role>",
+		Args:    cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return workspaceTeamAdd(cmd, out, args)
+		},
+	}
+	cmd.PersistentFlags().StringVar(&workspaceID, "workspace-id", "", "workspace to add the team to")
+	cmd.PersistentFlags().StringVar(&workspaceRole, "role", houston.WorkspaceViewerRole, "workspace role assigned to team")
+	return cmd
+}
+
+func newWorkspaceTeamUpdateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update team role",
+		Short: "Update a Team's Role for a Workspace",
```

Comment:
This should probably be `Update a Team inside a workspace` since the command is generic (update) with a --role flag to update the role